src/components/MainComponent.js

```diff
@@ -70,15 +70,26 @@ function Admin(props) {
     }
     return "Brand";
   };
+
+  const hideNavigationComponents = () => {
```

Comment:
this is a hack, why don't we put a check while rendering the navigation, it only gets rendered when a user is authenticated

Reply:
resolved